lib/travis/yml/schema/def/addon/pkg.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+require 'travis/yml/schema/type'
+
+module Travis
+  module Yml
+    module Schema
+      module Def
+        module Addon
+          class Pkg < Addon
+            register :pkg
+
+            def define
+              prefix :packages
+
+              map :packages, to: :seq, alias: :package, summary: 'Package names', eg: 'cmake'
```

Comment:
i'd suggest using `names` here `pkg` already suggests "package", so "package names" would read better than "package package". 

also, i'd suggest not adding aliases unless we have to.

Reply:
It was suggestion from freebsd community c (like apt)
```addons:
  pkg:
    packages:
    - cmake
    - vim
```
and alias
```addons:
  pkg:
    package: cmake
```

Reply:
On Ubuntu builds we keep aliases as follows:
```
addons:
  apt:
    packages:
```

- so FreeBSD community proposed something consistent. The `apt` is replaced with `pkg` in FreeBSD, which may be not so fortunate, but I'm for keeping it consistent with linux based mechanism - less confusion for users

Reply:
and we already made it consistent on macos:

```
addons:
  homebrew:
    packages:
```

so if we introduce something different now, we should allow it for all other OSes - I'd prefer to give less options to the users